pydm/widgets/slider.py

```diff
@@ -538,7 +543,9 @@ def calc_step_size(self):
         Given max, min, and num steps calculate step size
         """
         self.step_size = (self.maximum - self.minimum) / self.num_steps
```

Comment:
should we check for 0-division here?

could self.num_steps be 0 here when self.maximum == self.minimum? https://github.com/slaclab/pydm/blob/1b896f5579ea3f7899a49a25a0079445ee3be965/pydm/widgets/slider.py#L1020

Reply:
Yeah, it could. I'll put another check there.